### ISSUE TITLE
Loss function for 3D rotation instead of 1D rotation 

### ISSUE THREAD

**Issue by vinayver198** [charlesq34/frustum-pointnets#122]
Hi ,

I have a dataset that has 3 rotation angles , I have already done the requisite changes for training but facing some offsets when prediction the 3 angles (heading,pitch and roll) . To solve this problem you have introduced corner loss .
I believe i should modify the loss function to fix this.
Can @charlesq34  throw some light on this ?
